common/kubernetes-membership-scheme/src/main/java/org/wso2/carbon/membership/scheme/kubernetes/api/KubernetesHttpsApiEndpoint.java

```diff
@@ -92,4 +96,8 @@ public boolean verify(String hostname, SSLSession session) {
             HttpsURLConnection.setDefaultHostnameVerifier(dummyHostVerifier);
         } catch (Exception ignored) {}
     }
+
+    private String getServiceAccountToken() throws IOException {
+        return new String(Files.readAllBytes(Paths.get(KubernetesMembershipSchemeConstants.BEARER_TOKEN_FILE_LOCATION)));
```

Comment:
Thanks for the clarification @firzhan! Yes this is the recommended way to access the K8S API. Will merge the PR.